### PR TITLE
Step 5: Send movement page (for all movements) general view

### PR DESCRIPTION
**Pull request by awalkowiak**
https://tickets.pih-emr.org/browse/OBPIH-885

### CODE REVIEW

File: src/js/components/stock-movement-wizard/AddItemsPage.jsx [hunk 1]
```diff
@@ -4,7 +4,7 @@ import { reduxForm, initialize, formValueSelector } from 'redux-form';
 import { connect } from 'react-redux';
 import PropTypes from 'prop-types';
 
-import validate from './validate';
+import { validate } from './validate';
```
Comment (jmiranda) [inline src/js/components/stock-movement-wizard/AddItemsPage.jsx]: Can you explain what this does?

File: src/js/components/stock-movement-wizard/SendMovementPage.jsx [hunk 2]
```diff
@@ -1,34 +1,204 @@
...
+              </tr>
+            </thead>
+            <tbody>
+              {
+                _.map(
+                  tableItems,
+                  item =>
+                  (
+                    <tr key={item.lotWithBin}>
+                      <td key={item.lotWithBin + item.product.name}>{item.product.name}</td>
```
[the remark this answers is not in the document]
Comment (jmiranda) [reply src/js/components/stock-movement-wizard/SendMovementPage.jsx]: I'm not sure what the key is used for, but I assume the different keys allow you to reference row vs cell if necessary. A parent key would only allow you to reference the row-level component.

File: src/js/components/stock-movement-wizard/SendMovementPage.jsx [hunk 3]
```diff
@@ -1,34 +1,204 @@
...
+};
 
 const SendMovementPage = (props) => {
   const {
-    handleSubmit, pristine, previousPage, submitting,
+    handleSubmit, pristine, previousPage, submitting, pickPage, lineItems,
+    description, origin, destination, stockList, requestedBy, dateRequested,
   } = props;
+
+  console.log(props);
```
[the remark this answers is not in the document]
Comment (awalkowiak) [reply src/js/components/stock-movement-wizard/SendMovementPage.jsx]: oops, i forgot about it

File: src/js/components/stock-movement-wizard/SendMovementPage.jsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (awalkowiak) [reply src/js/components/stock-movement-wizard/SendMovementPage.jsx]: @pmuchowski i am not sure about it, ill check it out 
@jmiranda these keys are for React. Acording to docs: "Keys help React identify which items have changed, are added, or are removed. Keys should be given to the elements inside the array to give the elements a stable identity"

[the remark this answers is not in the document]
Comment (jmiranda) [reply src/js/components/stock-movement-wizard/SendMovementPage.jsx]: Got it. Thanks for the clarification.

File: src/js/components/stock-movement-wizard/AddItemsPage.jsx (hunk 1 above)
Comment (awalkowiak) [reply src/js/components/stock-movement-wizard/AddItemsPage.jsx]: This answer explains it the best: https://stackoverflow.com/a/36796281
Before this PR there was 'default export' in validate.js, and in this commit i made in this file 'named exports'

File: src/js/components/stock-movement-wizard/SendMovementPage.jsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (awalkowiak) [reply src/js/components/stock-movement-wizard/SendMovementPage.jsx]: @pmuchowski yup you are right, ill simplify this with index too

**Comment by awalkowiak** [comment]
@jmiranda fixed and rebased

File: src/js/components/stock-movement-wizard/AddItemsPage.jsx (hunk 1 above)
Comment (jmiranda) [reply src/js/components/stock-movement-wizard/AddItemsPage.jsx]: And ... that's why I hate javascript.

**Comment by jmiranda** [comment]
Requested changes have been commited so I'm going to dismiss Pawel's review so I can merge.

**Pull request merged**